Library/Formula/grass.rb

```diff
@@ -1,42 +1,34 @@
-require 'formula'
-
 class Grass < Formula
-  homepage 'http://grass.osgeo.org/'
-  revision 1
+  homepage "http://grass.osgeo.org/"
+  url "http://grass.osgeo.org/grass70/source/grass-7.0.0.tar.gz"
+  mirror "https://mirrors.kernel.org/debian/pool/main/g/grass/grass_7.0.0.orig.tar.gz"
+  sha256 "1324556514cc38aced1c6dce27933ce386be027b1e2594f807f67250fb55436a"
 
-  stable do
-    url "http://grass.osgeo.org/grass64/source/grass-6.4.4.tar.gz"
-    sha1 "0e4dac9fb3320a26e4f640f641485fde0323dd46"
+  head "https://svn.osgeo.org/grass/grass/trunk"
 
-    # Patches that files are not installed outside of the prefix.
-    patch :DATA
-  end
-
-  head do
-    url "https://svn.osgeo.org/grass/grass/trunk"
-
-    patch do
-      url "https://gist.githubusercontent.com/jctull/0fe3db92a3e7c19fa6e0/raw/42e819f0a9b144de782c94f730dbc4da136e9227/grassPatchHead.diff"
-      sha1 "ffbe31682d8a7605d5548cdafd536f1c785d3a23"
-    end
+  # prevent installer trying to write to /Library
+  patch do
+    url "https://gist.githubusercontent.com/dunn/776ccb55d8d4535798bf/raw/288f43b37f7b1016efcefaea37ef5133d93c75b0/grass.diff"
+    sha256 "c7020dd282e0144bd900d0ac5d51ada486c071d3826db5405d2b6ef63ee2d7fc"
   end
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!